packages/ketcher-core/src/domain/entities/PolymerBond.ts

```diff
@@ -78,6 +81,14 @@ export class PolymerBond extends DrawingEntity {
   }
 
   public get isSideChainConnection() {
+    if (
+      (this.secondMonomer instanceof Chem ||
+        this.secondMonomer instanceof Peptide) &&
+      this.firstMonomer instanceof RNABase
```

Comment:
The order of monomers can be opposite (Rna base can be second monomer).
Also we need to handle Rna base connections with all types of monomers. So if Rna base connection is not with sugar through r1(base) -> r3(sugar) then it should be interpreted as side connection